yogstation/code/modules/mob/living/simple_animal/drone/verbs.dm

```diff
@@ -0,0 +1,14 @@
+/////////////////////////
+//YOG DRONE VERBS//
+////////////////////////
```

Comment:
You never include this file in the .dme?

Reply:
Fixed